docs/FAQ.md

```diff
@@ -1,52 +1,134 @@
 
-We chose Github by the fact that it is the tool that we are most familiar with and that will naturally
-lead to integrations for Github specifically being developed in an early stage.
+### My company doesn't have thousands of developers or services. Is Backstage overkill?
 
-Hosting this project on Github does not exclude integrations with other alternatives such as Gitlab or
-Bitbucket. We believe that in time there will be plugins that will provide functionality for these tools
-as well. Hopefully contributed by the community.
+Not at all! A core reason to adopt Backstage is to standardize how software is built at your company. It's easier to decide on those standards as a small company, and grows in importance as the company grows. Backstage sets a foundation, and an early investment in your infrastructure becomes even more valuable as you grow.
 
-And note that implementations of Backstage can be hosted wherever you feel suits your needs best.
+### Our company has a strong design language system/brand that we want to incorporate. Does Backstage support this?
 
-## Q: Can Backstage by used for other things than developer portals?
+Yes! The Backstage UI is built using Material-UI. With the theming capabilities of Material-UI, you are able to adapt the interface to your brand guidelines.
 
-Yes. 
+## Technical FAQ:
 
-The core frontend framework could be used for building any large-scale web application where multiple teams are building separate parts of the app, but you want the overall experience to be consistent.
+### Why Material-UI?
 
-That being said, in [Phase 2](https://github.com/spotify/backstage#project-roadmap) of the project we will add features that are needed for developer portals and systems for managing software ecosystems. Our ambition will be to keep Backstage modular.
+The short answer is that's what we've been using in Backstage internally.
 
-## Q: My company doesn't have thousands of developers. Is Backstage overkill?
+The original decision was based on Google's Material Design being a thorough, well thought out and complete design system, with many mature and powerful libraries implemented in both the system itself and auxiliary components that we knew that we would like to use.
 
-Not really. Sure, having something like Backstage gets more important as the number of developers in your company grows. One of the core reasons to adopt Backstage is to help standardise how software is built at your company. Setting guidelines and deciding on standards is easier when your company is smaller.
+It strikes a good balance between power, customizability, and ease of use. A core focus of Backstage is to make plugin developers productive with as few hurdles as possible. Material-UI lets plugin makers get going easily with both well-known tech and a large flora of components.
+​
+### What framework does Backstage use?
```

Comment:
```suggestion
### What technology does Backstage use?
```

Neither of the things mentioned are frameworks, except maybe react, that's an ongoing discussion :grin: